lib/src/utf8.dart

```diff
@@ -31,15 +31,19 @@ class Utf8 extends Struct {
 
   /// Creates a [String] containing the characters UTF-8 encoded in [string].
   ///
-  /// The [string] must be a zero-terminated byte sequence of valid UTF-8
-  /// encodings of Unicode scalar values. A [FormatException] is thrown if the
-  /// input is malformed. See [Utf8Decoder] for details on decoding.
+  /// Either the [string] must be zero-terminated or its [length] must be
```

Comment:
Utf8 is a variable length encoding. Is this the `length` in encoded bytes? Or the number of characters when decoded?

Reply:
It would be the number of one-byte characters, same as `strlen()`. I have added a mention to the docs.